subiquity/client/controllers/filesystem.py

```diff
@@ -85,6 +90,9 @@ async def _wait_for_probing(self):
                       self.ui.body)
 
     def make_guided_ui(self, status):
+        se = self.storage_encryption = status.storage_encryption
+        if se is not None and se.support == StorageEncryptionSupport.DEFECTIVE:
+            1/0  # should show an error page here
```

Comment:
Adding a note as a reminder to come back to this.  Even throwing an Exception with a message would be a good placeholder.